docs/antrea-network-policy.md

```diff
@@ -272,6 +274,15 @@ single port, to the 10.0.10.0/24 subnet specified by the `ipBlock` field.
 **Note**: The order in which the egress rules are set matter, i.e. rules will
 be enforced in the order in which they are written.
 
+**enableLogging**: ClusterNetworkPolicy ingress or egress rule can be
+audited by enabling it's logging field. When `enableLogging` field is set to
+true, any packet (first packet in the connection) that is matched by this rule
```

Comment:
I would suggest: "the first packet of any connection that matches this rule"

Reply:
done